sources/api/early-boot-config/src/provider/aws.rs

```diff
@@ -87,25 +86,22 @@ impl PlatformDataProvider for AwsDataProvider {
 
         let mut client = ImdsClient::new();
 
-        // Attempt to read from local file first on the `aws-dev` variant
-        #[cfg(bottlerocket_platform = "aws-dev")]
-        {
-            match local_file_user_data()? {
-                None => warn!("No user data found via local file: {}", USER_DATA_FILE),
-                Some(s) => output.push(s),
-            }
+        // Attempt to read from local file first
+        match local_file_user_data()? {
+            Some(s) => output.push(s),
+            None => warn!("No user data found via local file: {}", USER_DATA_FILE),
         }
 
         // Instance identity doc next, so the user has a chance to override
         match Self::identity_document(&mut client).await? {
-            None => warn!("No instance identity document found."),
             Some(s) => output.push(s),
+            None => warn!("No instance identity document found."),
```

Comment:
Here I moved the `None` below the `Some` to make these `match`es uniform with the other providers.